index.rb

```diff
@@ -144,6 +146,38 @@ def show(**kwargs)
   end
 end
 
+venues << Venue.new(:name => 'Kilowatt', :link => 'https://kilowattbar.com/') do
+  URI.open(URI.join(link, 'events-1')) do |html|
+    # We need to pull the api key from a script tag inside an iframe. Nokogiri
+    # doesn't parse javascript, so a greasy regex will have to do for now.
+    Nokogiri::HTML(html).css('iframe').attr('srcdoc').text.match(/"apiKey":"(?<apikey>\w+)"/) do |match|
+      kw_api_key = match[:apikey]
+
+      if kw_api_key
```

Comment:
Are there cases where this won't be present? I don't think we want to silently fail if this stops working for some reason. I think I would just do `request['x-api-key'] = match[:apikey]` below.

Reply:
Yeah good call. Optimism is a core value of this project and these if blocks just don't jive with that. We can add explicit error handling as we encounter issues with this parser later on.